test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -397,6 +397,51 @@ def check_envelope(prototypes_base)
     end
   end
 
+  # Implement multiple LPD handling from userdata by space, space type and default space_type
+  #
+  # @param prototypes_base [Hash] Baseline prototypes
+  def check_multi_lpd_handling(prototypes_base)
+    prototypes_base.each do |prototype, model_baseline|
+      building_type, template, climate_zone, user_data_dir, mod = prototype
+      if user_data_dir == 'no_user_data'
+        sub_prototypes_base = {}
+        sub_prototypes_base[prototype] = model_baseline
+        check_lpd(sub_prototypes_base)
+      else
+        if user_data_dir == 'userdata_lpd_01'
+          space_name_to_lpd_target = {}
+          space_name_to_lpd_target['Attic'] =15.06948107
```

Comment:
Do you have a markdown file for documentation? Maybe we can document the basis for the expected values in the markdown file. That way if they fail in the future due to some change in either the prototypes or the standards, we can more easily debug the failures.

Reply:
Will add this in the markdown file, a new section called Test Cases.